github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -502,7 +502,7 @@ presubmits:
       fork-per-release: "true"
     always_run: true
     optional: false
```

Comment:
and optional remains false ?
its wont be visible why the PR isn't merged, or once skip_report=true it would auto make it not a gating ?

Reply:
Good catch. Fixed